components/Nav/Navbar/index.tsx

```diff
@@ -1,36 +1,204 @@
-import { classNames } from '~/utils/helpers';
-import ThemeSwitcher from './ThemeSwitcher';
-import VersionToggle from './VersionToggle';
-
-import RevisitOnboard from '/public/img/general/onboard-revisit.svg';
+import HelpIconSVG from '/public/img/general/onboard-revisit.svg';
 
 const NavBar: React.FC<{
     setShowOnboardModal?: React.Dispatch<React.SetStateAction<boolean>>;
 }> = ({ setShowOnboardModal }) => {
+    return (
+        <NavContextProvider>
+            <NavBarContent setShowOnboardModal={setShowOnboardModal} />
+        </NavContextProvider>
+    );
+};
+
+const NavBarContent: React.FC<{
+    setShowOnboardModal?: React.Dispatch<React.SetStateAction<boolean>>;
+}> = ({ setShowOnboardModal }) => {
+    const { navMenuOpen, setNavMenuOpen, launcherMenuOpen, setLauncherMenuOpen } = useContext(NavContext);
+    const [navBackdrop, setNavBackdrop] = useState<boolean>(true);
+    const routes = useRouter().asPath.split('/');
+    const route = routes[1];
+    const { account, network } = useStore(selectWeb3Info, shallow);
+
+    const listItemStyles = 'flex';
+    const linkStyles =
+        'flex transition-all items-center px-4 py-2 text-base cursor-pointer whitespace-nowrap dark:text-white text-tracer-650';
+    const selectedStyles = 'font-bold';
+
+    const handleScroll = (scrollContainer: HTMLDivElement) => {
+        if ((!!scrollContainer && scrollContainer.scrollTop >= 1) || window.innerWidth < 1024) {
+            setNavBackdrop(true);
+        } else {
+            setNavBackdrop(false);
+        }
+    };
+
+    useEffect(() => {
+        const scrollContainer = document.getElementById('__next') as HTMLDivElement;
+        handleScroll(scrollContainer);
+        scrollContainer && scrollContainer.addEventListener('scroll', () => handleScroll(scrollContainer));
+        return () => {
+            scrollContainer && scrollContainer.removeEventListener('scroll', () => handleScroll(scrollContainer));
+        };
+    }, [handleScroll]);
+
+    const handleMenuOpen = () => {
+        if (launcherMenuOpen) {
+            setLauncherMenuOpen(false);
+            setTimeout(() => {
+                setNavMenuOpen(true);
+            }, 550);
+        } else {
+            setLauncherMenuOpen(false);
+            setNavMenuOpen(true);
+        }
+    };
+
+    const handleLauncherOpen = () => {
+        if (navMenuOpen) {
+            setNavMenuOpen(false);
+            setTimeout(() => {
+                setLauncherMenuOpen(true);
+            }, 550);
+        } else {
+            setNavMenuOpen(false);
+            setLauncherMenuOpen(true);
+        }
+    };
+
+    const handleMenuClose = useCallback(() => {
+        setNavMenuOpen(false);
+    }, []);
+
+    const handleLauncherClose = useCallback(() => {
+        setLauncherMenuOpen(false);
+    }, []);
+
+    // Close nav after hitting desktop breakpoint
+    const handleResize = () => {
+        if (window.innerWidth > 767) {
```

Comment:
try re-use defined breakpoint sizing here

Reply:
used constant variable from BreakpointEnum in place of this